service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala

```diff
@@ -337,7 +332,7 @@ object FabricCryptoMaterial {
                 UseForEncipherment,
                 UseForClientAuth,
                 UseForServerAuth,
-                AlternativeDNSName(s"$componentName.$organization"),
+                AlternativeDNSName(s"$componentName"),
```

Comment:
just remove, it's literally the same as next line

Reply:
fixed.